Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicRequestBus.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+namespace AzToolsFramework
+{
+    namespace Prefab
+    {
+        class PrefabPublicRequests
+            : public AZ::EBusTraits
+        {
+        public:
+            using Bus = AZ::EBus<PrefabPublicRequests>;
+
+            //////////////////////////////////////////////////////////////////////////
+            // EBusTraits overrides
+            static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single;
+            //////////////////////////////////////////////////////////////////////////
+
+            virtual ~PrefabPublicRequests() = default;
+
+            /**
+            * Create a prefab out of the entities provided, at the path provided, and save it in disk immediately.
+            * Automatically detects descendants of entities, and discerns between entities and child instances.
+            */
+            virtual bool CreatePrefabInDisk(
```

Comment:
This is not required to be exposed in this bus right? We want prefabs to be only created in memory through the tests. And we don't want people to be using this ebus for regular prefab requests.

Reply:
Fixed.